front/src/App.tsx

```diff
@@ -1,35 +1,36 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useState } from "react";
+import "./App.css";
+import { Spell } from "./components/table/types";
```

Comment:
```suggestion
import type { Spell } from "./components/table/types";
```

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html